### ISSUE TITLE
how to use kagerou with WebSocket plugin?

### ISSUE THREAD

**Issue by pastarice** [hibiyasleep/kagerou#144]
I tried to input the url into websocket the same way that I do for mopimopi, but it doesn't work.

Would you please let me know how to set it up?

Thanks

**Comment by hibiyasleep** [comment]
this url should work: `http://unsecure.idyllshi.re/kagerou/overlay?HOST_PORT=ws://localhost:10501`

the `HOST_PORT` part is which should be attached by OverlayProc, but it seems didn't.

**Comment by FX8350** [comment]
I found several URLs. What's the difference?

1: `http://unsecure.idyllshi.re/kagerou/overlay?HOST_POR=ws://localhost:10501`
(posed on this thread)
2: `https://idyllshi.re/kagerou/overlay/?HOST_PORT=ws://127.0.0.1:10501/`
(https://ngld.github.io/OverlayPlugin/streamers)
3: `https://hibiyasleep.github.io/kagerou/overlay/?HOST_PORT=ws://127.0.0.1:10501/`
(NGLD OverlayPlugin 0.11.6.0 show this URL)
4: `http://kagerou.hibiya.moe/overlay/?HOST_PORT=ws://127.0.0.1:10501/`
(Redirected from `github.io` )

**Comment by hibiyasleep** [comment]
tl;dr: ACTWebSocket doesn't allow HTTPS for overlays since local WebSocket connection is not secured.

* `hibiyasleep.github.io` is the first and original URL served directly from GitHub. HTTPS is forced.
* `kagerou.hibiya.moe` is HTTP-only replacement; when WebSocket user opens HTTPS URL from its OverlayProcess, browser etc, an overlay will redirect user here.
* `idyllshi.re` is just yet another domain. I'm trying to make new users use this URL.
* `unsecure.idyllshi.re` exists for the same reason as `kagerou.hibiya.moe`.

**Comment by dkolb** [comment]
Not to necro this issue but I'm trying to set up to use my phone as a second screen.  `http://unsecure.idyllshi.re/kagerou/overlay?HOST_PORT=ws://{MY_IP}:10501` redirects me to `https://hibiya.moe/kagerou/overlay?HOSTPORT=ws://{MY_IP}:10501` and gives me NGINX Not Found.  This is for both Firefox and Chrome on Android 10.

**Comment by hibiyasleep** [comment]
I just failed to reproduce on Chrome on Android 9, but it could be my server configuration error... is `http://unsecure.~~~` URL working now?

**Comment by dkolb** [comment]
Chrome on Android 10 here.  It's still sending me to `https://hibiya.moe~~~`
